src/ms-store-badge.d.ts

```diff
@@ -13,6 +13,10 @@ declare class MSStoreBadge extends HTMLElement {
      * The ID of your app.
      */
     productId: string;
+    /**
+     * The optional campaign ID of your app.
+     */
+    cid: string;
```

Comment:
Nit: can we name this `campaignId`? We have inconsistent naming and casing here, "productId" vs "cid"